README.md

```diff
@@ -461,17 +461,24 @@ application.
 #### Implementing live preview in your application
 
 To set up live preview, listen for update events from the SDK. These events are triggered after content is
-edited in Kontent.ai, providing you with the updated data:
+edited in Kontent.ai, providing you with the updated data. 
+In a typical application, you would fetch the data from the Delivery API and store them in memory.
+When the SDK triggers an update event, you would then update the stored items in memory to display the latest content.
+To easily apply the updates on you items, you can use `applyUpdateOnItem` or `applyUpdateOnItemAndLoadLinkedItems` functions from the SDK.
 
 ```ts
-import KontentSmartLink, { KontentSmartLinkEvent } from '@kontent-ai/smart-link';
+import KontentSmartLink, { KontentSmartLinkEvent, applyUpdateOnItem, applyUpdateOnItemAndLoadLinkedItems } from '@kontent-ai/smart-link';
 
 // Initialize the SDK
 const sdk = KontentSmartLink.initialize({ ... });
 
 // Listen for updates and apply them to your application
 sdk.on(KontentSmartLink.Update, (data: IUpdateMessageData) => {
-  // Use this data to update your application state or UI as needed
+  // Use this data to update your application state or UI as needed e.g.:
+  setItems((items) => items.map(item => applyUpdateOnItem(item, data)));
+  // or
+  Promise.all(items.map(item => applyUpdateOnItemAndLoadLinkedItems(item, data, fetchItemsFromDeliveryApi)))
```

Comment:
don't we want to add below? 
I know its up to anybody how he want to get the items, but this example would save me 2 or 3 minutes till i figured out how to do it :D
```ts
const fetchItemsFromDeliveryApi = (items) => client.items().inFilter(system.codename, items).toAllPromise().then(res => res.data.items)

```